### ISSUE TITLE
Оптимизация функционала Подготовка и загрузка данных

### ISSUE THREAD

**Issue by akademik1986** [Pr-Mex/vanessa-automation#1561]
Укажите:
   1. Версия Vanessa Automation 1.2.38.1
   2. Версия платформы 8.3.20.1674
   3. Режим совместимости базы, где запускается Vanessa Automation 8.3.16
   4. Операционная система win10

На текущий момент функционал подготовки и загрузки данных документов состоит из следующих шагов
_И я проверяю или создаю для документа "ПоступлениеТоваровУслуг" объекты:
И я перезаполняю для объекта табличную часть "Оборудование":_
Первый шаг проводит документ поступления, если он выгружается с признаком Проведен=Истина, хотя его ТЧ еще пустая, и проведение происходит бесполезно.
Следующий шаг заполняет ТЧ документа, но происходит уже только запись документа.
Итого мы имеем проведение документа, его запись, а в итоге документ без движений все равно и в конце приходится еще кодом обыгрывать проведение ВСЕХ документов.
Предлагают убрать проведение документа на первом шаге, а генерировать отдельный шаг, завершающий заполнение документа, по проведению документа, если документ грузится с признаком проведен.

**Comment by DitriXNew** [comment]
Да. Такое есть. Думаем как в один шаг запихнуть сразу и тч. Но пока руки не дошли.

**Comment by akademik1986** [comment]
@DitriXNew  извиняюсь, по этому issue пока нет продвижений?
